pkg/resources/standard_database.go

```diff
@@ -0,0 +1,701 @@
+		schema.TypeString,
+		"The database parameter that specifies the default catalog to use for Iceberg tables.",
+		func(inner *schema.Schema) {
+			inner.ValidateDiagFunc = IsValidIdentifier[sdk.AccountObjectIdentifier]()
+		},
+	),
+	"replace_invalid_characters": nestedProperty(
+		schema.TypeBool,
+		"Specifies whether to replace invalid UTF-8 characters with the Unicode replacement character (�) in query results for an Iceberg table. You can only set this parameter for tables that use an external Iceberg catalog.",
+	),
+	"default_ddl_collation": nestedProperty(
+		schema.TypeString,
+		"Specifies a default collation specification for all schemas and tables added to the database. It can be overridden on schema or table level. For more information, see [collation specification](https://docs.snowflake.com/en/sql-reference/collation#label-collation-specification).",
+	),
+	"storage_serialization_policy": nestedProperty(
+		schema.TypeString,
+		fmt.Sprintf("Specifies the storage serialization policy for Iceberg tables that use Snowflake as the catalog. Valid options are: %v. COMPATIBLE: Snowflake performs encoding and compression of data files that ensures interoperability with third-party compute engines. OPTIMIZED: Snowflake performs encoding and compression of data files that ensures the best table performance within Snowflake.", sdk.AsStringList(sdk.AllStorageSerializationPolicies)),
+	),
+	"log_level": nestedPropertyWithInnerModifier(
+		schema.TypeString,
+		fmt.Sprintf("Specifies the severity level of messages that should be ingested and made available in the active event table. Valid options are: %v. Messages at the specified level (and at more severe levels) are ingested. For more information, see [LOG_LEVEL](https://docs.snowflake.com/en/sql-reference/parameters.html#label-log-level).", sdk.AsStringList(sdk.AllLogLevels)),
+		func(inner *schema.Schema) {
+			inner.DiffSuppressFunc = func(k, oldValue, newValue string, d *schema.ResourceData) bool {
+				return strings.EqualFold(oldValue, newValue) || (d.Get(k).(string) == string(sdk.LogLevelOff) && newValue == "")
+			}
+		},
+	),
+	"trace_level": nestedProperty(
+		schema.TypeString,
+		fmt.Sprintf("Controls how trace events are ingested into the event table. Valid options are: %v. For information about levels, see [TRACE_LEVEL](https://docs.snowflake.com/en/sql-reference/parameters.html#label-trace-level).", sdk.AsStringList(sdk.AllTraceLevels)),
+	),
+	"replication": {
+		Type:        schema.TypeList,
+		Optional:    true,
+		Description: "Configures replication for a given database. When specified, this database will be promoted to serve as a primary database for replication. A primary database can be replicated in one or more accounts, allowing users in those accounts to query objects in each secondary (i.e. replica) database.",
+		MaxItems:    1,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"enable_for_account": {
```

Comment:
I left it as one because in HCL is treated as an element, so it's used as
```terraform
replication_config {
  enable_for_account{} # 1st item
  enable_for_account{} # 2nd item
}
```
But I can change it if we want to

Reply:
My mistake then. Optionally you can change `for` -> `to` just like in the docs.